.gitignore

```diff
@@ -42,3 +42,5 @@ local.properties
 # Sandbox stuff
 _sandbox
 
+signing.properties
+droidconit.jks
```

Comment:
I think `signing.properties` _should_ be in the `.gitignore` as it's named in the `build.gradle` and must never be committed